tensorflow/tools/docker/Dockerfile

```diff
@@ -29,8 +30,8 @@ RUN pip --no-cache-dir install \
 
 # Install TensorFlow CPU version.
 ENV TENSORFLOW_VERSION 0.7.1
-RUN pip --no-cache-dir install \
-    http://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-${TENSORFLOW_VERSION}-cp27-none-linux_x86_64.whl
+ADD whl/tensorflow-*.whl /tmp/
```

Comment:
Unfortunately it's difficult to parameterize Dockerfiles. How about this, I can move this content into Dockerfile.devel (replacing the existing ones) and restore the http URL in this file?